lib/actions/local/index.ts

```diff
@@ -1,4 +1,4 @@
-###
```

Comment:
I bundled this file in as a typescript conversion because it's so tiny and in conjunction with https://github.com/balena-io/balena-cli/pull/1754 will remove all coffeescript from lib/actions/local